src/freenas/etc/ix.rc.d/ix-autotune

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+#
+# $FreeBSD$
+#
+#
+
+# PROVIDE: ix-autotune
+# REQUIRE: middlewared
+# BEFORE: NETWORKING
+
+. /etc/rc.freenas
+
+do_autotune()
+{
+	/usr/local/bin/midclt call system.advanced.autotune sysctl
+	/usr/local/bin/midclt call system.advanced.autotune loader
```

Comment:
Just noticed the return code is the result of the call as text, not the actual exit code of the process